autogoal/contrib/ensemble/_stacking.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+from sklearn import svm
+from sklearn.naive_bayes import GaussianNB
+
+
+class StackingEnsemble:
+
+    def __init__(self, X, y):
+       self.network = []
+       self.X = X
+       self.y = y
+
+    def layer_constructor(self,  estimators = ['svm', 'gnb']):
```

Comment:
Ideally, these methods should be internal, and the user would never need to call them manually. Also, instead of defining the models by name, we can just pass an instance of the right algorithm, since all `sklearn` estimators have `fit` and `predict`. 

I imagine something like this:

```
ensemble = StackingEnsemble(
    layers = [
        [SVC(), LogisticRegression()],
        [DecisionTreeClassifier()]
    ],
    final = GaussianNB()
)

ensemble.fit(X, y)
# ...
```

This way the user instantiates the internal algorithms, passes them to the stacking ensemble, and just has to call `fit`.